Classes/common/CDTFetchChanges.m

```diff
@@ -39,6 +54,14 @@ - (instancetype)initWithDatastore:(CDTDatastore *)datastore
 
 - (void)main
 {
+    // Ensure our settings are not changed during execution
+    self.mDocumentChangedBlock = [self.documentChangedBlock copy];
+    self.mDocumentWithIDWasDeletedBlock = [self.documentWithIDWasDeletedBlock copy];
+    self.mFetchRecordChangesCompletionBlock = [self.fetchRecordChangesCompletionBlock copy];
```

Comment:
The new properties specify the attribute `copy`, I do not think you need to copy the original block before assigning it (check [apple doc](https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html), under title *Copy Properties Maintain Their Own Copies*)